examples/transformers_text.py

```diff
@@ -172,20 +219,15 @@ def forward(self, feat: dict[str, MultiNestedTensor]) -> Tensor:
         out = self.model(input_ids=input_ids, attention_mask=mask)
 
         # Return value has the shape [batch_size, 1, text_model_out_channels]
-        if self.pooling == "mean":
-            return mean_pooling(out.last_hidden_state, mask)
-        elif self.pooling == "cls":
-            return out.last_hidden_state[:, 0, :].unsqueeze(1)
-        else:
-            raise ValueError(f"{self.pooling} is not supported.")
+        return mean_pooling(out.last_hidden_state, mask)
```

Comment:
Can we add that we are using mean-based pooling in the doc-string of `TextToEmbeddingFinetune`?

Reply:
Sure